.github/workflows/windows.yml

```
@@ -415,42 +415,49 @@ jobs:
   msvc:
     name: 'msvc (${{ matrix.arch }}, ${{ matrix.plat }}, ${{ matrix.name }})'
     runs-on: windows-latest
-    timeout-minutes: 30
+    timeout-minutes: 55
     strategy:
       matrix:
         include:
           # name should be unique and without special chars. It creates a folder with that name for the cache.
         - name: 'schannel'
-          install: 'zstd zlib nghttp2 libssh2[core,zlib]'
+          install: 'brotli libpsl zstd zlib pkgconf gsasl nghttp2 libssh2[core,zlib]'
           arch: 'x64'
           plat: 'windows'
           type: 'Debug'
-          tflags: '~1516 ~2301 ~2302 ~2303 ~2307'
-          config: '-DENABLE_DEBUG=ON -DENABLE_UNICODE=ON  -DCURL_USE_SCHANNEL=ON  -DCURL_ZSTD=ON -DUSE_NGHTTP2=ON -DBUILD_SHARED_LIBS=OFF -DCURL_USE_LIBSSH2=ON'
+          tflags: '~1516 ~2301 ~2302 ~2303 ~2307 ~165 ~1448 ~2046 ~2047'
+          config: '-DENABLE_DEBUG=ON -DENABLE_UNICODE=ON  -DCURL_USE_SCHANNEL=ON  -DCURL_BROTLI=ON -DCURL_USE_LIBPSL=ON -DCURL_ZSTD=ON -DUSE_NGHTTP2=ON -DCURL_USE_GSASL=ON -DBUILD_SHARED_LIBS=OFF -DCURL_USE_LIBSSH2=ON -DUSE_WIN32_IDN=ON'
         - name: 'openssl'
-          install: 'zstd zlib nghttp2 nghttp3 openssl libssh2'
+          install: 'brotli libpsl zstd zlib pkgconf gsasl nghttp2 nghttp3 openssl libssh2'
           arch: 'x64'
           plat: 'windows'
           type: 'Debug'
           tflags: '~1516 ~2301 ~2302 ~2303 ~2307'
-          config: '-DENABLE_DEBUG=ON -DENABLE_UNICODE=OFF -DCURL_USE_SCHANNEL=OFF -DCURL_ZSTD=ON -DUSE_NGHTTP2=ON -DBUILD_SHARED_LIBS=OFF -DCURL_USE_LIBSSH2=ON -DCURL_USE_OPENSSL=ON -DUSE_OPENSSL_QUIC=ON'
+          config: '-DENABLE_DEBUG=ON -DENABLE_UNICODE=OFF -DCURL_USE_SCHANNEL=OFF -DCURL_BROTLI=ON -DCURL_USE_LIBPSL=ON -DCURL_ZSTD=ON -DUSE_NGHTTP2=ON -DCURL_USE_GSASL=ON -DBUILD_SHARED_LIBS=OFF -DCURL_USE_LIBSSH2=ON -DUSE_LIBIDN2=ON -DCURL_USE_OPENSSL=ON -DUSE_OPENSSL_QUIC=ON'
+        - name: 'openssl'
+          install: 'brotli zstd zlib nghttp2 nghttp3 openssl libssh2'
+          arch: 'x64'
+          plat: 'uwp'
+          type: 'Debug'
+          tflags: 'skipall'
+          config: '-DENABLE_DEBUG=ON -DENABLE_UNICODE=OFF -DCURL_USE_SCHANNEL=OFF -DCURL_BROTLI=ON -DCURL_ZSTD=ON -DUSE_NGHTTP2=ON -DBUILD_SHARED_LIBS=OFF -DCURL_USE_LIBSSH2=ON -DUSE_LIBIDN2=ON -DCURL_USE_OPENSSL=ON -DUSE_OPENSSL_QUIC=ON'
```

```suggestion
          config: '-DENABLE_DEBUG=ON -DENABLE_UNICODE=OFF -DCURL_USE_SCHANNEL=OFF -DCURL_BROTLI=ON -DCURL_USE_LIBPSL=ON -DCURL_ZSTD=ON -DUSE_NGHTTP2=ON -DCURL_USE_GSASL=ON -DBUILD_SHARED_LIBS=OFF -DCURL_USE_LIBSSH2=ON -DCURL_USE_OPENSSL=ON -DUSE_OPENSSL_QUIC=ON'
        - name: 'openssl'
          install: 'brotli zstd zlib nghttp2 nghttp3 openssl libssh2'
          arch: 'x64'
          plat: 'uwp'
          type: 'Debug'
          tflags: 'skipall'
          config: '-DENABLE_DEBUG=ON -DENABLE_UNICODE=OFF -DCURL_USE_SCHANNEL=OFF -DCURL_BROTLI=ON -DCURL_ZSTD=ON -DUSE_NGHTTP2=ON -DBUILD_SHARED_LIBS=OFF -DCURL_USE_LIBSSH2=ON -DCURL_USE_OPENSSL=ON -DUSE_OPENSSL_QUIC=ON'
```
libidn2 is not available for Windows in vcpkg. This can hopefully be fixed and we can give this another go.